### ISSUE TITLE
Add issue templates for requesting new repository & adding new contributor

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#6298]
Add new issue templates to support the following tasks - similar to enhancement, bug, release
 see https://github.com/odpi/egeria-docs/issues/217 for requesting new repository
 see https://github.com/odpi/egeria-docs/issues/347 for adding new contributors
 
 The templates will capture required info & provide a checklist of tasks to complete

**Comment by mandy-chessell** [comment]
OK to have templates for requesting a new repo - but I would think the voting process involved in adding new contributors/maintainers should remain private to the maintainers?

**Comment by planetf1** [comment]
Suggestion
 - we create a new private mailing list for Egeria maintainers only
 - this is the list used for proposing a new maintainer etc
 - a github template can help with the checklist for the actions to take once the vote is complete

**Comment by planetf1** [comment]
Initial template for requesting a new repo has been added - to gather info from author, and to provide a checklist to work through. Will refine as we go through adding an entry for the new strimzi connector.

**Comment by planetf1** [comment]
Review comments to be actioned in next iteration of template - from PR https://github.com/odpi/egeria/pull/6325

- [ ] template refers to egeria docs 'Add link in egeria docs' -- this should be a link
- [ ] under 'Release' note that the release process for the new repo should be updated in usual place (add link)
- [ ] The assumptions is a long list, maybe better as a link to process docs. My response - we don't have the process docs yet. Checklist is a quick start. Will raise issue or add here to complete those docs

**Comment by planetf1** [comment]
Review comments to be actioned in next iteration of template - from PR https://github.com/odpi/egeria/pull/6325

- [ ] template refers to egeria docs 'Add link in egeria docs' -- this should be a link
- [ ] under 'Release' note that the release process for the new repo should be updated in usual place (add link)
- [ ] The assumptions is a long list, maybe better as a link to process docs. My response - we don't have the process docs yet. Checklist is a quick start. Will raise issue or add here to complete those docs

**Comment by planetf1** [comment]
As the steps for adding a new contributor are fairly limited, and also since there may be some discussion on a private list (as for maintainers), it doesn't seem necessary to add a new template for this. So closing. See referenced issues for doc updates.
